docker/confd/templates/behat.yml.tmpl

```diff
@@ -40,18 +40,6 @@ default:
                 - DigidepsBehat\FeatureContext: [{ dbName: "api", sessionName: 'digideps' }]
             {{ end }}
 
-        admin:
-            description: End to end journey for Admin user
-            paths:    [ %paths.base%/features/admin ]
-            filters:
-                tags: "~@javascript"
-            contexts:
-            {{ if exists "/api/database/name" }}
-                - DigidepsBehat\FeatureContext: [{ dbName: {{ getv "/api/database/name" }} }]
-            {{ else }}
-                - DigidepsBehat\FeatureContext: [{ dbName: "api" }]
-            {{ end }}
-
```

Comment:
Is this removing the admin suite?

Reply:
No, the definition's in the twice for some reason so I removed one of them.